tools/cicd/integration-tests.yaml

```diff
@@ -3,10 +3,8 @@ steps:
   # Check if the Docker image for Python 3.11 exists
   - name: 'gcr.io/cloud-builders/docker'
     id: 'check_python_image'
-    entrypoint: 'bash'
```

Comment:
bash -c is the "legacy" way of running bash scripts.
Better to use `script` which avoid the issue with the substitution of Cloud Build variables.

See: https://cloud.google.com/build/docs/configuring-builds/run-bash-scripts#running_inline_bash_scripts_legacy